src/client/components/atoms/burgerButton/index.tsx

```diff
@@ -0,0 +1,52 @@
+import './styles.scss';
+import { useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import StateContext from '../../../utils/stateContext';
+import ExpandMenu from '../collapse/expandMenu';
+
+const BurgerButton = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { user } = useContext(StateContext);
+
+  const handleToggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+  return (
+    <>
```

Comment:
We normally use a react-fragment to wrap several sibling elements, but never as `Block` container.

suggestion: 

- use a div with a block className
- tweak the rest of classNames to be aligned with BEM